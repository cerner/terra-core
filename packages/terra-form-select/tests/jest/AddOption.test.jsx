import React from 'react';
/* eslint-disable-next-line import/no-extraneous-dependencies */
import { shallowWithIntl } from 'terra-enzyme-intl';
import AddOption from '../../src/shared/_AddOption';

describe('AddOption', () => {
  it('should render a default AddOption', () => {
    const wrapper = shallowWithIntl(<AddOption value="value" />).dive();
    expect(wrapper).toMatchSnapshot();
  });
});
