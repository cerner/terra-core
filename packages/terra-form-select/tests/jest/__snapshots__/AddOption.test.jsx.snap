// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddOption should render a default AddOption 1`] = `
<Option
  data-terra-select-add-option={true}
  disabled={false}
  display="Terra.form.select.add"
  intl={
    Object {
      "defaultFormats": Object {},
      "defaultLocale": "en",
      "formatDate": [Function],
      "formatHTMLMessage": [Function],
      "formatMessage": [Function],
      "formatNumber": [Function],
      "formatPlural": [Function],
      "formatRelative": [Function],
      "formatTime": [Function],
      "formats": Object {},
      "formatters": Object {
        "getDateTimeFormat": [Function],
        "getMessageFormat": [Function],
        "getNumberFormat": [Function],
        "getPluralFormat": [Function],
        "getRelativeFormat": [Function],
      },
      "locale": "en",
      "messages": null,
      "now": [Function],
      "onError": [Function],
      "textComponent": "span",
      "timeZone": null,
    }
  }
  isAddOption={true}
  value="value"
/>
`;
